client/src/components/Common/Typography/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Typography, ConfigProvider } from 'antd';
+import PropTypes from 'prop-types';
+
+import 'antd/dist/antd.css';
+
+import './style.css';
+
+const { Title, Paragraph } = Typography;
```

Comment:
can you add a comment to tell us how can we use this component beside how we can change the font

Reply:
We use Arabic font, and I didn't find support for Arabic font In browsers
- we must use a default font 
@ahmadysalah